tests/spark_config_test.py

```diff
@@ -220,6 +223,45 @@ def test_get_mesos_docker_volumes_conf(
         )
         assert sorted(output[validate_key].split(',')) == sorted(set(expected_volumes))
 
+    @pytest.mark.parametrize(
+        'volumes', [
+            None,
+            [
+                {'hostPath': '/host/file1', 'containerPath': '/containter/file1', 'mode': 'RO'},
+                {'hostPath': '/host/file2', 'containerPath': '/containter/file2', 'mode': 'RO'},
+                {'hostPath': '/host/paasta1', 'containerPath': '/container/paasta1', 'mode': 'RO'},
+            ],
+        ],
+    )
+    @pytest.mark.usefixtures('mock_existed_files')
+    def test_get_k8s_docker_volumes_conf(self, volumes):
+        expected_volumes = {}
+
+        def _get_k8s_volume(host_path, container_path, mode, count=itertools.count()):
```

Comment:
I don't think we should duplicate an api definition inside the test file. 
Can this be imported and used? Probably, we can define this outside `_get_k8s_docker_volumes_conf` api.